janus_core/stats.py

```diff
@@ -0,0 +1,69 @@
+"""
+Module that reads the md stats output timeseries.
+"""
+
+import re
+
+from numpy import genfromtxt
+
+from janus_core.janus_types import PathLike
+
+
+class Stats:
+    """
+    Configure shared molecular dynamics simulation options.
+
+    Parameters
+    ----------
+    source : PathLike
+        File that contains the stats of a molecular dynamics simulation.
+    """
+
+    def __init__(self, source: PathLike) -> None:
+        """
+        Initialise MD stats reader.
+
+        Parameters
+        ----------
+        source : PathLike
+            File that contains the stats of a molecular dynamics simulation.
+        """
+
+        self.rows = 0
+        self.columns = 0
+        self.data = None
+        self.labels = None
+        self.units = None
```

Comment:
Would these things be better as `@property`s? Do we expect them to be manually set?

Reply:
yes

Reply:
done